src/platforms/python/common/configuration/integrations/gql.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: gql
+description: gql GraphQL client integration
+sidebar_order: 55
+---
+
+The [gql](https://gql.readthedocs.io/en/stable/) integration allows you to monitor all GraphQL requests that your Python application makes using the `gql` library. Specifically, this integration reports every [`TransportQueryError`](https://gql.readthedocs.io/en/latest/modules/transport_exceptions.html#gql.transport.exceptions.TransportQueryError) to Sentry and provides additional context including the URL the request was made to, GraphQL query which caused the error, and the `errors` information that the GraphQL server returns.
```

Comment:
```suggestion
The [GQL](https://gql.readthedocs.io/en/stable/) integration allows you to monitor all GraphQL requests that your Python application makes using the `gql` library. Specifically, this integration reports every [`TransportQueryError`](https://gql.readthedocs.io/en/latest/modules/transport_exceptions.html#gql.transport.exceptions.TransportQueryError) to Sentry and provides additional context including:
- The URL the request was made to
- The GraphQL query which caused the error
- The `errors` information that the GraphQL server returns
```

Reply:
suggest capitalization fix and breaking up features into a bulleted list for readability